wiki/datasets/traffic-modelling-datasets.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Traffic Modelling Datasets
+published: true
+---
+
+Traffic modelling is a hot topic in the field of autonomous cars currently. Here you will find a list of open datasets which can be used as source for building a traffic model. The list will include data captured from a variety of sources as listed below:
+- UAV/Drones
+- Traffic Camera 
+- Autonomous Cars
+
+> This is not a list of datasets for learning-to-drive. This list is more focused towards dataset which provide global perspective of the traffic scenarios, rather than ego-vehicle perspective. Though few ego-vehicle datasets can be used for traffic modelling as well.
+
+Each dataset is scored based upon several parameters crucial for learning behavior and interactions between vehicles in a recorded scenario.
+
+<iframe height="100%" width="100%" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vRNtQVog9cMhYrOuv0fGlEKyK4HiuwD0orLoXEfO9z-Jtz9hvEB6qzdZy6lOSsyyPGakYe6ggYBAi8u/pubhtml?gid=0&amp;single=true&amp;widget=true&amp;headers=false"></iframe>
```

Comment:
This spreadsheet should be here and not linked to a Google doc. The spreadsheet also appears to be incomplete.